locations/linked_data_parser.py

```diff
@@ -183,22 +183,27 @@ def parse(response, wanted_type, json_parser="json") -> Feature:
     #  "opens":  "09:00:00"
     # }
     # See https://schema.org/OpeningHoursSpecification for further examples.
+    #
+    # This defaults to DAYS_EN, which is the standard (see https://schema.org/DayOfWeek)
+    # As some publishers publish localised days, we allow this to be overridden.
     @staticmethod
-    def _parse_opening_hours_specification(oh: OpeningHours, rule: dict, time_format: str):
+    def _parse_opening_hours_specification(oh: OpeningHours, rule: dict, time_format: str, days: {} = DAYS_EN):
         if (
             not type(LinkedDataParser.get_case_insensitive(rule, "dayOfWeek")) in [list, str]
             or not type(LinkedDataParser.get_case_insensitive(rule, "opens")) == str
             or not type(LinkedDataParser.get_case_insensitive(rule, "closes")) == str
         ):
             return oh
 
-        days = LinkedDataParser.get_case_insensitive(rule, "dayOfWeek")
-        if not isinstance(days, list):
-            days = [days]
+        parsed_days = LinkedDataParser.get_case_insensitive(rule, "dayOfWeek")
+        if not isinstance(parsed_days, list):
+            parsed_days = [parsed_days]
 
-        for day in days:
+        for day in parsed_days:
+            # Handle plain text days, or URI references to a DayOfWeek enumeration, ie https://schema.org/Friday
+            parsed_day = day.strip().replace("http://schema.org/", "").replace("https://schema.org/", "")
             oh.add_range(
-                day=day.strip(),
+                day=days[parsed_day],
```

Comment:
This is the main bit I am not sure about; as before this just took a string - including "https://schema.org/Friday" - and presumably tolerated that. Now we're validating against DAYS_EN.